app/views/users/show.html.erb

```diff
@@ -5,8 +5,9 @@
       <div class="mid-col-4"><%= image_tag @user.avatar_url(250)%></div>
       <div class="mid-col-4"></div>
     <small>email: <%= @user.email%>,</small>
-    <small>count of lists <%= @user.lists.count%></small>
-    <h2>To-do Lists </h2>
-    <p><%= render @user.lists %></p>
+    <small>count of lists <%= @user.items.count%></small>
+    <h2>To-do Items </h2>
+    <p><%= render "items/item", item: @user.items %></p>
```

Comment:
related to my comment above, you are passing a collection of items (plural) to a variable named `item` (singular).  Also.. you probably don't want to nest a list of elements inside a `<p>` tag.  This is intended for a paragraph of text